### DISCUSSION TITLE
[Bug]: "Config" is deprecated and will be removed in HA Core 2025.11

### DISCUSSION THREAD

**Discussion by ErikvO** [blakeblackshear/frigate#14857] [Report a Bug]
### Checklist
- [X] I have updated to the latest available Frigate version.

### Describe the problem you are having
Home Assistant 2024.11.0 log shows:
```
Logger: homeassistant.core
Source: helpers/deprecation.py:222
First occurred: 8:02:46 AM (1 occurrences)
Last logged: 8:02:46 AM
Config was used from frigate, this is a deprecated alias which will be removed in HA Core 2025.11. Use homeassistant.core_config.Config instead, please report it to the author of the 'frigate' custom integration
```

### Steps to reproduce
1. Have frigate installed
2. Update to Home Assistant 2024.11.0
3. Get deprecation message in the Home Assistant logs

### Version
0.14.1-f4f3cfa

### Operating system
HassOS

### Install method
HassOS Addon

**Comment by tbrasser** [comment]
Need to import Config from core_config instead of from core.

**Comment by NickM-27** [comment]
This is already fixed and this is the wrong repo to report this in

**Reply by ErikvO** [reply]
You’re right. Sorry for the confusion.
I was surprised I couldn’t find a bug report already reporting this issue. Now it all makes sense: wrong repository 🤦‍♂️
